docs/FAQ.rst

```diff
@@ -0,0 +1,286 @@
+Q&A
+***
+
+Q: How do I know if DALI can help me?
+#####################################
+A: You need to check our docs first and see if DALI operators cover your use case. Then, try to run
+a couple of iterations of your training with a fixed data source - generating the batch once and
+reusing it over the test run to see if you can train faster without any data processing. If so,
+then the data processing is a bottleneck, and in that case, DALI may help. This topic is covered
+in detail in
+`the GTC'22 talk <https://www.nvidia.com/gtc/session-catalog/#/session/1636559250287001p4DG>`_.
+
+Q: What data formats does DALI support?
+#######################################
+A: DALI can load most of the image file formats (JPEG, PNG, TIFF, JPEG2000 and more) as well
+as audio (WAV, OGG, FLAC) and video. The files can be stored individually or grouped in storage
+containers from common DL frameworks - TFRecord, RecordIO, caffe/caffe2 LMDB. WebDataset is
+also supported. In case there's no native support for a particular format, the data can be
+loaded and parsed in user-provided python code and supplied via external source.
+
+Q: How does DALI differ from TF, PyTorch, MXNet, XYZ FW
+#######################################################
+A: The main difference is that the data preprocessing, and augmentations are GPU accelerated,
+and the processing is done for the whole batch at the time to improve GPU utilization. Also,
+it can be used in multiple different FW - TF, MXNet, PyTorch, PaddlePaddle - so you are sure
+that the data processing is bit-exact, which is important when you move models between FWs
+and see some discrepancies in the accuracy. Moreover, it provides a convenient way to switch
+data processing between GPU and CPU to utilize all available computational resources.
+
+Q: What to do if DALI doesn't cover my use case?
+################################################
+A: You can experiment first with writing a custom operator in Python to check if you can create
+a data pipeline you like (you can even use CuPy inside to utilize GPU). Then you can create
+a plugin with an operator running the native code or use Numba to convert python code to
+a native operator. You can also become a contributor and make your work a part of
+the DALI code base - we would be more than happy to review and accept any PR with new
+functionality.
+
+Q: How to use DALI for inference?
+#################################
+A: You can easily employ DALI for inference together with the TRITON server. We developed
+a dedicated DALI backend so all the user needs to do is provide a description of the processing
+pipeline to the TRITON server, and add DALI into the model ensemble.
```

Comment:
How about putting some links here?
```suggestion
A: You can easily employ DALI for inference together with the `Triton Inference Server <https://github.com/triton-inference-server>`_. 
We developed a dedicated `DALI Backend <https://github.com/triton-inference-server/dali_backend>`_ 
so all you need to do is to provide a description of the processing pipeline to the Triton, and add 
DALI to the model ensemble. For more information about using DALI with Triton, please refer to the 
`DALI Backend documentation <https://github.com/triton-inference-server/dali_backend#how-to-use>`_
```

Reply:
Fixed, I think the `https://developer.nvidia.com/nvidia-triton-inference-server` link is more appropriate than github one.